host_os.py

```diff
@@ -32,11 +34,36 @@
     'build-iso': build_iso,
 }
 
+def is_package_installed(package_name):
+    """
+    Checks if a RPM package is installed
+    """
+
+    cmd = "rpm -q %s" % package_name
+    try:
+        run_command(cmd, shell=True)
+    except SubprocessError as e:
+        # grep returns 1 when search string is not found and 2 if an error occurred
+        if e.returncode == 1:
+            return False
+        else:
+            raise
+
+    return True
+
 
 if __name__ == '__main__':
     CONF = config.setup_default_config()
     subcommand = CONF.get('default').get('subcommand')
 
+    PACKAGE_DEPENDENCIES_FILE_PATH = "rpm_requirements.txt"
+    package_dependencies = open(PACKAGE_DEPENDENCIES_FILE_PATH).readlines()
+    package_dependencies = [x.strip('\n') for x in package_dependencies]
+    for package in package_dependencies:
+        if not is_package_installed(package):
+            print("%s package should be installed before running this script" % package)
+            sys.exit(4)
+
```

Comment:
I agree with @mopsfelder, just adding that `readlines` does not return line separators, so the `.strip('\n')` could be removed.